site/assets/css/fish_style.css

```diff
@@ -4,7 +4,12 @@ body {
     line-height: 20px;
 }
 
-img, video {
+video {
+    width: 100% !important;
+    height: 100% !important;
```

Comment:
Why the `!important` here?

I would try to avoid it if at all possible. If it's specificity you can give these videos a class or even id if you want.

Reply:
Okay I can't seem to find a difference here at all.

How do you get this to change the video's size compared to before?

Reply:
Hi, this was for the new video. I am gonna create a high-res video today, I tried with a different video file with high res and I forgot to delete !important tags.

We can remove these important tags now, in next commit I'll fix the video.

I was trying to make this full size of the page:

![image](https://github.com/user-attachments/assets/c16e1e6e-d2ea-461f-9427-9c4c1043f53e)

Reply:
The video is pretty low res because I want it to be very small in terms of filesize.

So we don't really *want* a "high res" video here because we want the site to work with very very low resource usage.

Reply:
The images have also grown by quite a lot.

Have you tried running them through optipng?

Reply:
@faho I can recommend using WebP format for images if you want, that's the best practice so far for quality and compression at the same time.

Instead of using optipng to compress them, the WebP method can also work.

If PNGs are mandatory, I can do optipng and commit it.

Other than that, what do you think of using animated codeblocks, just like the ones in docs page but animated kind of with javascript instead of video. This way it could increase readability in small screens and big screens too while still keeping it low-resource? 

![image](https://github.com/user-attachments/assets/8aaa85f1-0c44-4e52-8842-7afde71830ab)

Reply:
> animated kind of with javascript 

We try not to add more javascript here.

> using WebP format for images

webP should be fine by now.

Reply:
Hi @faho,

I've added the new commit with the updates.

The CSS turned into old style: 

```css
img, video {
    max-width: 100%;
    max-height: fit-content;
    border: 0;
    margin: 0 auto 0;
}
```

Also, the images are converted to WebP format and updated in HTML code.